s4e-web/cypress/page-objects/auth/auth-login.po.ts

```diff
@@ -0,0 +1,128 @@
+import { Core } from '../core.po';
+
+export interface User {
+    email: string;
+    password: string;
+}
+
+export class Login extends Core {
+    static readonly pageObject = {
+        getLoginInput: () => cy.get('input[data-e2e="login-email-input"]'),
+        getPasswordInput: () => cy.get('input[data-e2e="login-password-input"]'),
+        getSubmitBtn: () => cy.get('button[data-e2e="login-submit-btn"]'),
+        getGoToMapBtn: () => cy.get('a[data-e2e="go-to-map-btn"]'),
+        getOptionsBtn: () => cy.get('[data-e2e="loginOptions-btn"]'),
+        getLoginOutBtn: () => cy.get('[data-e2e="logout-btn"]'),
+        getOptionsDropdown: () => cy.get('[data-e2e="options-dropdown"]', { timeout: 15000 }),
+
+        getFieldErrors: () => cy.get('.invalid-feedback > .ng-star-inserted'),
+        getError: () => cy.get('.message')
+    };
+
+    static fillForm(user: User) {
+        Login
+            .pageObject
+            .getLoginInput()
+            .type(user.email);
+        Login
+            .pageObject
+            .getPasswordInput()
+            .type(user.password);
+
+        return Login;
+    }
+
+    static unfillForm() {
+        Login
+            .pageObject
+            .getLoginInput()
+            .click()
```

Comment:
In here you should use .clear() instead of .click()

Reply:
replace name `unfillfirm` to `touchFormfields`

Reply:
Second thing is that this same behaviour you can achive be clicking `Zaloguj` btn, when fields are in any state, so in this case I'd like to ask you to remove the function. In tests run just `sendForm`.

Reply:
I cant use only senFom, I have to touch the field if I want to check displaying of error message, so I can't remove this function.